src/main/cpp/mechanisms/ClimberManager/decoratormods/ClimberManager.cpp

```diff
@@ -78,8 +80,18 @@ void ClimberManager::Update(RobotStateChanges::StateChange change, int value)
 {
 	if (change == RobotStateChanges::ClimbModeStatus)
 		m_climbMode = static_cast<RobotStateChanges::ClimbMode>(value);
+	if (change == RobotStateChanges::GameState)
+		m_gamePeriod = static_cast<RobotStateChanges::GamePeriod>(value);
 }
 
+bool ClimberManager::isClimbMode()
+{
+	return m_climbMode == RobotStateChanges::ClimbMode::ClimbModeOn;
+}
+bool ClimberManager::IsEnabled()
+{
+	return m_gamePeriod != RobotStateChanges::GamePeriod::Disabled;
+}
 // todo not sure what to do with this
```

Comment:
if you're not implementing these; delete the commented out code

Reply:
I need this, but I am not implementing it in this story. Since it isn't about the state transitions